unyt/array.py

```diff
@@ -125,6 +125,7 @@
     UnitConversionError,
     UnitsNotReducible,
     SymbolNotFoundError,
+    UnitOperationWarning,
```

Comment:
Though import ordering isn't automatically enforced atm, can you please reorder this one to preserve alphabetical order here ?